package.json

```
@@ -22,6 +22,7 @@
     "jest": "26.0.1",
     "jsonlint2": "1.7.1",
     "npm-run-all": "4.1.5",
+    "svg-path-bounding-box": "^1.0.4",
```

To avoid a PR by [Renovate](https://docs.renovatebot.com/) that pins this dependency, please update this to:

```suggestion
    "svg-path-bounding-box": "1.0.4",
```